test/lib/foreman/renderer_test.rb

```diff
@@ -18,6 +18,17 @@ def setup_safemode_renderer
     assert_nothing_raised(NoMethodError) { foreman_url }
   end
 
+  test "pxe_kernel_options are not set when no OS is set" do
+    @host = FactoryGirl.create(:host)
+    assert_empty pxe_kernel_options
```

Comment:
`[]` is also empty, I'd suggest using `assert_equal '', pxe_kernel_options` instead to properly test the return value for this issue.